gradle.properties

```diff
@@ -4,24 +4,24 @@ pluginGroup = intellij
 pluginName = intellij-buf
 pluginRepositoryUrl = https://github.com/bufbuild/intellij-buf
 # SemVer format -> https://semver.org
-pluginVersion = 0.4.3
+pluginVersion = 0.5.0
 
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
-pluginSinceBuild = 223
-pluginUntilBuild = 243.*
+pluginSinceBuild = 243
+pluginUntilBuild = 253.*
 
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
 platformType = IU
-platformVersion = 2022.3
+platformVersion = 2024.3
```

Comment:
Should this be 2025.1 instead?

Reply:
2025.1 doesn't exist yet to build against. This won't be available until it is out of EAP.

Reply:
Switched this to 2024.2 - this will ensure we build/compile against the lowest supported version.